security/compliance_operator/compliance-operator-release-notes.adoc

```diff
@@ -13,6 +13,30 @@ These release notes track the development of the Compliance Operator in the {pro
 
 For an overview of the Compliance Operator, see xref:../../security/compliance_operator/compliance-operator-understanding.adoc#understanding-compliance-operator[Understanding the Compliance Operator].
 
+[id="compliance-operator-release-notes-0-1-59"]
+== OpenShift Compliance Operator 0.1.59
+
+The following advisory is available for the OpenShift Compliance Operator 0.1.59:
+
+* link:https://access.redhat.com/errata/RHBA-2022:8538[RHBA-2022:8538 - OpenShift Compliance Operator bug fix update]
+
+
+[id="compliance-operator-0-1-59-new-features-and-enhancements"]
+=== New features and enhancements
+
+The Compliance Operator now supports Payment Card Industry Data Security Standard (PCI DSS) `ocp4-pci-dss` and `ocp4-pci-dss-node` profiles on all architectures.
```

Comment:
We could clarify this by saying specifically `ppc64le` and `s390x`, which are the architectures that were recently added for PCI-DSS.

This will provide more clarity for the reader since we've supported PCI-DSS on `x86_64` for a while.

Reply:
Updated! Removed s390x since  s390x is exclude from PCI-DSS architectures